<?xml version="1.0"?>
<launch>
  <arg name="world_file" default="$(find moby_stage)/worlds/land_rover.world"/>
  <param name="/use_sim_time" value="true"/>

  <!--  ================ Stage ================  -->
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(arg world_file)">
    <param name="base_watchdog_timeout" value="0.5"/>
    <remap from="/cmd_vel" to="/car/cmd_vel"/>
    <remap from="/base_pose_ground_truth" to="/car/base_pose_ground_truth"/>
  </node>


  <!-- ========= solabot linear navigation ========= -->
  <node name="solabot_commands" type="solabot_teleop_joy.py" pkg="moby_stage" output="screen"/>
  <node pkg="joy" type="joy_node" name="joystick" >
    <param name="autorepeat_rate" value="100" />
  </node>

</launch>
